src/plugins/index_pattern_editor/public/components/empty_prompts/empty_index_pattern_prompt/empty_index_pattern_prompt.tsx

```diff
@@ -47,36 +39,43 @@ export const EmptyIndexPatternPrompt = ({
     >
       <EuiFlexGroup gutterSize="xl" alignItems="center" direction="rowReverse" wrap>
         <EuiFlexItem grow={1} className="inpEmptyIndexPatternPrompt__illustration">
-          <Illustration />
+          <Suspense fallback={<div>...loading...</div>}>
```

Comment:
I addressed this without DelayRender - Its unclear to me how DelayRender should be used. This is a large (by pixel count) image that won't disrupt flow so I just assume display it to users as fast as possible. I'm not sure where we'd benefit from a delay.

Reply:
I think I've mistaken `DelayRender` with `DelayHide` https://eui.elastic.co/#/utilities/delay

```
This avoids UI glitches that are common with loading spinners and other elements that are rendered conditionally and potentially for a short amount of time.
```

The idea is to avoid UI glitches when using a spinner.
So if we want, we can wrap a `<EuiLoadingSpinner size="xl" />` into it. But I agree that it might not be a good idea for an illustration use case